model/project_ref.go

```diff
@@ -507,12 +505,12 @@ func FindTaggedProjectRefs(includeDisabled bool, tags ...string) ([]ProjectRef,
 
 // FindAllMergedTrackedProjectRefs returns all project refs in the db
 // that are currently being tracked (i.e. their project files
-// still exist)
+// still exist and the project is not hidden)
 func FindAllMergedTrackedProjectRefs() ([]ProjectRef, error) {
 	projectRefs := []ProjectRef{}
 	err := db.FindAll(
 		ProjectRefCollection,
-		bson.M{ProjectRefTrackedKey: true},
+		bson.M{ProjectRefHiddenKey: false},
```

Comment:
The existing data in prod hasn't yet been changed to have hidden:false for all project refs, so we can't make this change yet. I don't see that Annie has written a migration script yet, so if you'd like to do it I can help you with it

Reply:
Just updated the query